docs/building-a-checkout-path.md

```diff
@@ -0,0 +1,73 @@
+## The Basics
+
+A checkout flow consists of a flow controller (some code that decides what step you are on and which steps are complete and incomplete) and several checkout step components. At the end of a checkout flow, your goal is to place a valid order using the `placeOrder` GraphQL mutation. This documentation goes in hand with the [Storefront guide](../guides/storefront-guide.md) to help you create a compelling experience in your storefront.
+
+## Collect an email address
+If the shopper is logged in, you may choose to skip this step and use one of the emails from their account. You can also move this step to somewhere else in the flow as long as you have an email address by the time you create the order. An order cannot be created without an email address.
+
+### Example Storefront Component Library components
+The `GuestForm` React component can be used to collect an email address for anonymous (guest) checkout.
+
+### GraphQL
+Use the `setEmailOnAnonymousCart` mutation if you have a cart and want to store the provided email address on it. This is recommended for data completeness and to avoid re-entry if the checkout flow is interrupted. However, you could also just cache this data on the client until you call the place order mutation.
+
+## Display fulfillment groups and optionally allow moving items among them
+To create an order, each item must be in exactly one [fulfillment group](../developers-guide/concepts/fulfillment-groups.md). There can be any number of fulfillment groups, but at a minimum there must be one per unique combination of fulfillment type and shop ID.
```

Comment:
Probably need to figure out what to do with these links to docs that don't exist yet. Also need to figure out what the linking syntax is for inter-document links since this doesn't seem to be what is currently in use.

Reply:
will remove for now